plugins/discovery-file/build.gradle

```diff
@@ -53,6 +54,7 @@ integTestCluster {
     dependsOn setupSeedNodeAndUnicastHostsFile
     clusterName = 'discovery-file-test-cluster'
     extraConfigFile 'discovery-file/unicast_hosts.txt', srcUnicastHostsFile
+    module project(':modules:discovery-zen')
```

Comment:
No, that's not true. This just ensures that the zen discovery module is installed as well on the deployed node for the REST test. The REST tests for plugins use the stripped down distribution `integ-test-zip`, not including the modules. The discovery-file module cannot be meaningfully tested without zen-discovery though.